aggregators/1.0/modules/ROOT/pages/aggregators-size-timeout.adoc

```diff
@@ -0,0 +1,154 @@
+= Configure Size-Based Aggregator Timeout and Max Size Fields - Mule 4
+
+To understand the Aggregators module *Size based aggregator* timeout behavior, review the following diagram that explains how the *Timeout* and *Max size* fields work. Then learn how to configure these fields in Anypoint Studio and in the XML editor in a Mule application example.
+
+Consider the following definitions:
+
+* The *Max size* field indicates the total number of elements to be aggregated before considering the aggregation complete.
+* The *Timeout* field indicates the maximum time to wait for the aggregation to complete.
+
+.Size-Based Aggregator Timeout and Max Size behavior diagram
+image::aggregator-size-timeout-diagram.png[]
+
+* The first process indicates that the aggregation is complete because the aggregated elements reached the total number of required elements before the timeout occurred.
+
+* The second process indicates that the aggregation is not complete because the aggregated elements did not reach the total number of required elements before the timeout occurred, which causes a reset of the aggregation.
+
+* The third process indicates that you can use an *Aggregator listener* source to capture the elements that were aggregated before the timeout occurred. The source triggers when the timeout occurs.
+
+If you choose to use an *Aggregator listener* source to capture incomplete aggregation elements, you must select the *Include timed out group* field to indicate that the source is triggered when the timeout occurs.
+
+== Configure the Mule Application
+
+The following example shows how to configure the *Size based Aggregator* scope to aggregate up to `3` HTTP request calls within `10` seconds. The application uses DataWeave to set the payload of each call with a random ID number. When the aggregation reaches the `3` calls required for completion and before the timeout occurs, the application logs all the aggregated elements. Otherwise, the application logs only the aggregated elements captured by an *Aggregator listener* source when the timeout occurred.
```

Comment:
```suggestion
The following example shows how to configure the *Size based aggregator* scope to aggregate up to `3` HTTP request calls within `10` seconds. The application uses DataWeave to set the payload of each call with a random ID number. When the aggregation reaches the `3` calls required for completion and before the timeout occurs, the application logs all the aggregated elements. Otherwise, the application logs only the aggregated elements captured by an *Aggregator listener* source when the timeout occurred.
```

Reply:
I'm not sure that we really need to format those numbers as syntax numerals....